packages/app/src/navigation/SidebarNavigation.cy.tsx

```diff
@@ -215,4 +240,75 @@ describe('SidebarNavigation', () => {
       cy.findByTestId('debug-badge').should('not.exist')
     })
   })
+
+  context('runs status icon', () => {
+    it('renders passing status if run status is "RUNNING" with no failures', () => {
+      mountComponent({ latestCloudProject: { status: 'RUNNING', numFailedTests: 0 } })
+      cy.findByTestId('icon-status-message').should('be.visible').contains('Latest run is in progress')
+      cy.percySnapshot('Runs Icon:running')
```

Comment:
I matched using percySnapshot to the above debug tests, but I wonder if it will work for `running` and `failing` as they are animated icons. Should we capture these, or do we have a way to disable animations globally for screenshots?

Reply:
Found this: https://gist.github.com/lunalium/6c7d689b1689b022b02f

Maybe useful!

Reply:
I'll check it out, though I might have been premature, I saw this on the percy doc site:
https://docs.percy.io/docs/animations#automatic-animation-freezing-in-percy